networkx/algorithms/tests/test_d_separation.py

```diff
@@ -154,9 +154,9 @@ def test_cyclic_graphs_raise_error():
     with pytest.raises(nx.NetworkXError):
         nx.d_separated(g, {0}, {1}, {2})
     with pytest.raises(nx.NetworkXError):
-        nx.minimal_d_separator(g, {0}, {1})
+        nx.minimal_d_separator(g, 0, 1)
```

Comment:
Ooo...  That's a good catch of a sneaky test error. The inputs aren't nodes, but it was raising an error before it ever checked if they were nodes. Still doing that, bu at least the input nodes are now valid nodes. :}

Reply:
Yep and at least the docstring is clear that the algorithm does not work for a set of nodes, so that is good